tests/asconfig/index.js

```diff
@@ -20,6 +22,15 @@ asc.main(["assembly/index.ts", "--outFile", "output.wasm", "--explicitStart", ..
     console.error("No binary was generated for the asconfig test in " + process.cwd());
     process.exit(1);
   }
+  const optionsPath = path.join(process.cwd(), "options.json")
+  if (fs.existsSync(optionsPath)) {
+    const options = require(optionsPath);
+    for (let option of Object.getOwnPropertyNames(options) ){
+      if (options[option] != asc.args[option]) {
```

Comment:
```suggestion
      if (options[option] !== asc.args[option]) {
```

Reply:
I'm skeptical this comparison is even necessary in the first place since it's essentially testing the options.merge() utility function.

Asserting some value in the binary is reasonable enough.